src/UI/SignTransactionsModals/SignWithDeviceModal/components/components/ConfirmAmount/components/ConfirmAmountNftSft/ConfirmAmountNftSft.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { MouseEvent } from 'react';
+import { faArrowUpRightFromSquare } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import classNames from 'classnames';
+
+import { WithStylesImportType } from 'hocs/useStyles';
+import { withStyles } from 'hocs/withStyles';
+import { AccountInfoSliceNetworkType } from 'types';
+import { NftEnumType } from 'types/tokens.types';
+import {
+  explorerUrlBuilder,
+  getExplorerLink
+} from 'utils/transactions/getInterpretedTransaction/helpers';
+
+export interface ConfirmAmountNftSftPropsType extends WithStylesImportType {
+  tokenAvatar: string;
+  ticker?: string;
+  network: AccountInfoSliceNetworkType;
+  identifier?: string;
+  name?: string;
+  type?: NftEnumType;
+  amount: string;
+}
+
+const ConfirmAmountNftSftComponent = ({
+  identifier,
+  tokenAvatar,
+  network,
+  ticker,
+  name,
+  styles,
+  amount,
+  type
+}: ConfirmAmountNftSftPropsType) => {
+  const isSft = NftEnumType.SemiFungibleESDT === type;
+  const duplicatedSftAvatars = Array(Math.min(4, Number(amount))).fill(null);
```

Comment:
Why 4?

Reply:
This is the amount of mirrored times the `SFT` preview is supposed to be displayed. 
<img width="402" alt="image" src="https://github.com/multiversx/mx-sdk-dapp/assets/24439772/27665585-e910-4f2c-ad3a-1cef95f9a5fd">